packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,8 +25,10 @@ const Breadcrumb = ({
   });
 
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav aria-label="Breadcrumb">
```

Comment:
```suggestion
    <nav className={className} aria-label="Breadcrumb" {...rest}>
```

Also, should `aria-label` be configurable?

Reply:
good catch